index.html

```diff
@@ -71,12 +71,283 @@ <h2 class="primary-text dark center-text">
+                <li class="tag flex-center">CSS</li>
+                <li class="tag flex-center">Javascript</li>
+                <li class="tag flex-center">Ruby on Rails</li>
+              </ul>
+              <button class="button card-button white">See project</button>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-image"></div>
+            <div class="card-info">
+              <h4 class="card-title title-text center-text dark">Multi-Post Stories Gain+Glory</h4>
+              <ul class="tag-container">
+                <li class="tag flex-center">HTML</li>
+                <li class="tag flex-center">CSS</li>
+                <li class="tag flex-center">Javascript</li>
+                <li class="tag flex-center">Ruby on Rails</li>
+              </ul>
+              <button class="button card-button white">See project</button>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-image"></div>
+            <div class="card-info">
+              <h4 class="card-title title-text center-text dark">Multi-Post Stories Gain+Glory</h4>
+              <ul class="tag-container">
+                <li class="tag flex-center">HTML</li>
+                <li class="tag flex-center">CSS</li>
+                <li class="tag flex-center">Javascript</li>
+                <li class="tag flex-center">Ruby on Rails</li>
+              </ul>
+              <button class="button card-button white">See project</button>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-image"></div>
+            <div class="card-info">
+              <h4 class="card-title title-text center-text dark">Multi-Post Stories Gain+Glory</h4>
+              <ul class="tag-container">
+                <li class="tag flex-center">HTML</li>
+                <li class="tag flex-center">CSS</li>
+                <li class="tag flex-center">Javascript</li>
+                <li class="tag flex-center">Ruby on Rails</li>
+              </ul>
+              <button class="button card-button white">See project</button>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-image"></div>
+            <div class="card-info">
+              <h4 class="card-title title-text center-text dark">Multi-Post Stories Gain+Glory</h4>
+              <ul class="tag-container">
+                <li class="tag flex-center">HTML</li>
+                <li class="tag flex-center">CSS</li>
+                <li class="tag flex-center">Javascript</li>
+                <li class="tag flex-center">Ruby on Rails</li>
+              </ul>
+              <button class="button card-button white">See project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="flex-center flex-column align-center">
+      <div class="aboutme section-container flex-center flex-column align-center">
```

Comment:
- [ ] Kindly consider adding the decoration background image seen in the figma design to make it look similar to the figma design.
your design
![fd](https://user-images.githubusercontent.com/96683843/234336878-30e0412d-e394-45d5-a4d0-e5a784219d7c.jpg)

figma design
![as](https://user-images.githubusercontent.com/96683843/234336714-620bcc5a-79ce-45a4-96f5-046edf2c3b3d.jpg)